rpc_proxy.go

```diff
@@ -201,6 +218,17 @@ func (p *rpcProxy) forwardGRPCRequest(resp http.ResponseWriter, req *http.Reques
 	p.grpcServer.ServeHTTP(resp, req)
 }
 
+func (p *rpcProxy) handleHealthCheckRequest(resp http.ResponseWriter, req *http.Request) {
+	status := statusServing
+	if p.tdexdConn == nil {
```

Comment:
is this enough to know is conected? 🤔

Reply:
Yes. The proxy always starts without a connection. The service attempts to establish a connection with the daemon at the given tdexdconnect URL only by calling its `/connect` endpoint.